Makefile

```diff
@@ -17,6 +17,9 @@ GO_ENVIRONMENTS ?=
 
 # Wait4X output name
 WAIT4X_BINARY_NAME ?= wait4x
+ifeq ($(GOOS), "windows")
+WAIT4X_BINARY_NAME ?= ${WAIT4X_BINARY_NAME}.exe
+endif
```

Comment:
I checked the output of pipeline, It doesn't work. https://github.com/atkrad/wait4x/suites/8894740761/artifacts/406904731

I have a suggestion, it will work:
```suggestion
ifeq ($(GOOS),windows)
WAIT4X_BINARY_NAME := ${WAIT4X_BINARY_NAME}.exe
endif
```

Reply:
ah perfect 👍 wasn't exactly sure about the syntax but will commit the suggestion